task/01-strings-tasks.js

```diff
@@ -238,7 +254,8 @@ function encodeToRot13(str) {
  *   isString(new String('test')) => true
  */
 function isString(value) {
-    throw new Error('Not implemented');
+    if (typeof value ==="string" || value instanceof String) return true;
```

Comment:
do not use pattern if(expression) return true else return false
(typeof value ==="string" || value instanceof String) is already a boolean value, just return it